mapping/sustainability.yaml

```
@@ -1714,16 +1714,17 @@
 
         For each meeting:
 
-        1. Publish the meeting agenda. Add a document, set its `.documentType` to 'lobbyingMeetingAgenda' and its `.url` to the URL at which the agenda is available.
-
-        2. Publish the meeting details. Add a `Meeting` object to the `.lobbyingMeetings` array and set:
+        1. Publish the meeting details. Add a `Meeting` object to the `.lobbyingMeetings` array and set:
 
         - `.id` incrementally
         - `.date` to the date of the meeting
         - `.address` to the address of the meeting
-        - `.beneficiary` to the name of the organisation or interest group that ultimately benefits from the lobbying activity
+        - `.numberOfParticipants` to the number of people present at the meeting
+        - `.publicOfficial.name` to the name of the person who was representing the organization at the meeting.
+        - `.publicOfficial.organization` to the `.name` and `.id` of the organization the public official represents. These should match the organization's entry in the `parties` array
+          ` `.publicOfficial.jobTitle`to the job title of the person named in`.name\`
 
-        3. Publish the meeting minutes. Add a document, set its `.documentType` to 'lobbyingMeetingMinutes' and its `.url` to the URL at which the meeting minutes are available.
+        2. Publish the meeting minutes. Add a document, set its `.documentType` to 'minutes.lobbyingMeeting' and its `.url` to the URL at which the meeting minutes are available.
```

Reference the common operation for adding a project document:

```suggestion
        2. Publish the meeting minutes. [Add a project document](../common.md#add-a-project-document) and set its `.documentType` to 'minutes.lobbyingMeeting'.
```